x-pack/plugins/alerting/server/lib/get_execution_log_aggregation.test.ts

```diff
@@ -216,6 +207,21 @@ describe('getExecutionLogAggregation', () => {
                       field: 'kibana.alert.rule.execution.metrics.number_of_generated_actions',
                     },
                   },
+                  numActiveAlerts: {
```

Comment:
I am wondering if we should match what we are going to do with our [rule saved object RFC](https://docs.google.com/document/d/1-tMHRS3liqhHf8YGVqR93rZg3zdoDe2nEfptkOw1Q-E/edit#heading=h.1w0d61eheso3). 

```
"alerts_count": {
    "active" 2,
    "new": 1,
    "recovered": 0,
    “ignored”: 0,
}
```

Reply:
Sure. I can update the `IExecutionLog` to this format

Reply:
Or are you asking if we should store it this way in the event log `execute` document?

Reply:
Done in this PR: https://github.com/elastic/kibana/pull/131848